### ISSUE TITLE
[linux] UI flickering/missing

### ISSUE THREAD

**Issue by laur89** [ferdium/ferdium-app#1499]
### Avoid duplicates

- [X] I agree to follow the [Code of Conduct](https://github.com/ferdium/ferdium-app/blob/HEAD/CODE_OF_CONDUCT.md) that this project adheres to.
- [X] I have searched the [issues tracker](https://github.com/ferdium/ferdium-app/issues?q=is%3Aopen+is%3Aissue+label%3A%22bug+%3Abeetle%3A%22) for a bug report similar to mine, in vain

### Ferdium Version

6.6.0

### What Operating System are you using?

Other Linux

### Operating System Version

Debian testing

### What arch are you using?

x64

### Expected Behavior

UI consistently working.

### Actual Behavior

Every now and then the UI is messed up - either parts of it are blank (as seen on screenshot) or flickering in and out of existence. Have seen this problem on other electron applications, e.g. Slack. With Slack there's a way out: `Help -> Troubleshooting -> Clear Cache and Restart`. Does similar mechanic exist for Ferdium; if not, could it be added?

### Steps to reproduce

No idea how to consistently reproduce this. This time it got sorted with upgrade to `6.7.0`. Note upgrade to `6.6.0` on the other hand did not resolve it.

### Screenshots

![image](https://github.com/ferdium/ferdium-app/assets/4551018/9771b412-18b2-497a-a7dc-7401c0993122)

**Comment by Cknight70** [comment]
Hello, are you running wayland? You can check by typing ``echo $XDG_SESSION_TYPE`` into your terminal

If this is the case, you can run ferdium natively with wayland which would work better. You can do this by adding these options to the end of your command to launch Ferdium. You can do this in your terminal, or add it to the end of your .desktop launch file startup command, or add it globally for your electron launch options in `~/.config/electron-flags.conf`

`--enable-features=WaylandWindowDecorations --ozone-platform-hint=auto`

You can also clear your cache by right clicking the service icon (the whats app icon on the far left) and selecting clear cache.

**Comment by laur89** [comment]
Nope, I'm still on x11, running [i3](https://github.com/i3/i3/) window manager.

**Comment by aquada** [comment]
I also have this issue (NixOS unstable Xorg) and resolved by installing flatpak version (6.6.0) and now working. I have resolved this in the past with other Electon apps (Joplin) by rm -R  ~/.config/<appname>/GPUCache, but this did not work for Ferdium

**Comment by laur89** [comment]
Ditto, tried nuking cache-sounding directories willy-nilly as a last resort (including GPUCache), but nothing did the trick.
